handlers.go

```diff
@@ -4,44 +4,102 @@ import (
 	"fmt"
 	"net/http"
 
+	"encoding/json"
 	"github.com/Financial-Times/go-fthealth/v1a"
-	"regexp"
+	log "github.com/Sirupsen/logrus"
```

Comment:
We're barely logging in the ingester anyway and deciding what logging libs to use across teams is a wider converations. I'd say leave it like this for now unless @tosan88 strongly objects.